src/mixins/filing-mixin.ts

```diff
@@ -0,0 +1,111 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { mapActions, mapState } from 'vuex'
+import { FilingData } from '@/interfaces'
+import { EntityTypes, FilingCodes, FilingTypes, FilingNames } from '@/enums'
+
+/**
+ * Mixin that provides some useful filing utilities.
+ */
+@Component({
+  computed: {
+    ...mapState(['filingData', 'entityType'])
+  },
+  methods: {
+    ...mapActions(['setFilingData'])
+  }
+})
+export default class FilingMixin extends Vue {
```

Comment:
Well done, i considered doing this as well but perhaps i was feeling the time crunch :(. 
Good job, thanks for doing this!